app/services/nsm/assessment_syncer.rb

```diff
@@ -20,11 +20,11 @@ def call
     private
 
     def sync_overall_comment
-      comment_event = app_store_record['events'].select { _1['public'] && _1['event_type'] == 'decision' }
-                                                .max_by { DateTime.parse(_1['created_at']) }
-
       return if claim.status.in?(%w[submitted granted])
 
+      event_type = claim.status == 'provider_requested' ? 'send_back' : 'decision'
```

Comment:
Two things here. First, because status is an enum, you can call `claim.provider_requested?` instead of doing a string comparison. Second, I think _both_ send-back states use the `send_back` event type, i.e. "further_info" as well as "provider_requested"

Reply:
updated the above however blocked by SNYK due to braces